wdio.conf.js

```diff
@@ -0,0 +1,182 @@
+// Stop node from complaining about fake memory leaks at higher concurrency
+require("events").defaultMaxListeners = 20;
+require("dotenv").config();
```

Comment:
Related to my earlier question, it's not super obvious to me why these need to be required before all those `const`s are declared.

Reply:
in the same vein, how would you feel about using this as an opportunity to start the switch to ESM?

Reply:
This is run via the node process, which doesn't support ESM without extra assistance

Reply:
right. and you're also right with the unsaid "adding the `babel-node` binary to this PR is out of scope."